openshift_cli_installer/libs/clusters/ipi_cluster.py

```diff
@@ -89,6 +92,11 @@ def _create_install_config_file(self):
             "pull_secret": self.pull_secret,
         }
 
+        platform = self.cluster_info["platform"]
+        if platform == GCP_STR:
```

Comment:
`platform` value is getting used in line 115 for getting config template, otherwise I'm getting

Reply:
`platform` value is getting used in line 115, and I'm also getting the below pre-commit error, if I add the suggestion, so I think we can keep the code as it is for this line.
 `F841 Local variable "platform" is assigned to but never used`

Reply:
when you use wolrus operator, it will assign the value to the variable.
you need to delete 
```
        platform = self.cluster_info["platform"]
```
assignment